docs/README.md

```diff
@@ -0,0 +1,157 @@
+# Documentation for Jenkins Chatbot Plugin
+
+This documentation folder contains resources and instructions related to the development of the RAG-based chatbot plugin for Jenkins.
+
+## Data Collection: Setup & Usage
+
+The data collection scripts are located under:
+
+```
+chatbot-core/data/collection/
+```
+
+These scripts gather information from three key sources:
+- Jenkins official documentation
+- Discourse community topics
+- StackOverflow threads
+
+### Setup Instructions
+
+> **Note**: Python **3.10.12** was used during development.  
+> Please use **Python 3.10.12 or later**. Ensure your installation includes support for the `venv` module.
```

Comment:
Pyhon 3.10 will be reach end of life next year (2026-10). Maybe recomend using 3.11 or later.